unit-tests/src/test/scala/scala/scalanative/posix/TimeTest.scala

```diff
@@ -133,6 +185,61 @@ class TimeTest {
     }
   }
 
+  @Test def strptimeDoesNotWriteMemoryOutsideStructTm(): Unit = {
+    Zone { implicit z =>
+      // Linux 56 Bytes, Posix specifies 36 but allows more.
+      val tmBufSize = 7
+      val tmBuf     = alloc[Ptr[Byte]](tmBufSize) // will zero/clear all bytes
+      val tmPtr     = tmBuf.asInstanceOf[Ptr[tm]]
```

Comment:
You just introduced a memory overflow. You allocated 7 bytes but using at 36 bytes.

Reply:
I believe that I was allocating 7 pointers of 8 bytes each, for a total of 56 bytes.

I put an assert against 56 bytes in a private copy of TimeTest.scala and it
supported my belief.

As a mark of respect for the many bugs you have discovered in Scala Native
I re-wrote that test to, I hope, make the memory allocation easier to verify and 
review.

Thank you for motivating the improvement.

Reply:
@LeeTibbert thanks, it was defently unclear code. Sorry for missunderstanding.